deployments/hadoop-yarn/ansible/35-config-zeppelin-interpreters.yml

```diff
@@ -20,20 +20,24 @@
 #
 #
 
-- name: "Configure Spark workers"
-  hosts: workers
+- name: "Configure Zeppelin interpreters"
+  hosts: zeppelin
   gather_facts: false
   vars_files:
     - config/ansible.yml
     - /tmp/ansible-vars.yml
 
   tasks:
 
-#    - name: "Create Spark temp directory"
-#      include_tasks: "tasks/create-linked.yml"
-#      vars:
-#        linkdest: "{{sptempdest}}"
-#        linkpath: "{{sptemplink}}"
-#        linkuser: "{{spuser}}"
+    - name: "Copy interpreter.json from deployment directory to Zeppelin conf"
+      copy:
+        src: "{{ playbook_dir | dirname | dirname }}/common/zeppelin/interpreter.json"
+        dest: "{{zephome}}/conf/interpreter.json"
 
 
+    - replace:
+        path: "{{zephome}}/conf/interpreter.json"
+        regexp: "^file:///home/fedora/.m2/repository$"
+        replace: "{{ zepmavendest }}"
+      delegate_to: zeppelin
+
```

Comment:
This feels really fragile.

* If someone reads the config file in deployments/common they would assume the maven directory is in the fedora user's home directory.
* If someone makes a tiny change to the value in the config file in deployments/common it won't match the regex expression any more.

We own the config file this comes from, `deployments/common/zeppelin/interpreter.json` , so why not edit the original config and treat it as a template?